plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/extension/MetalavaExtension.kt

```diff
@@ -98,4 +98,15 @@ open class MetalavaExtension {
      * The default is "current".
      */
     var releaseType = "current"
+
+    /**
+     * The directories to search for source files. An exception will be thrown is the named
+     * directories are not direct children of the project root. The default is "src".
+     */
+    var sourcePath = mutableSetOf<String>("src")
+
+    /**
+     * If the tasks should run as part of Gradle's `check` task. The default is yes.
+     */
+    var enforceCheck = true;
```

Comment:
As mentioned offline, I've not added any tests for these due to the current configuration. Once published, I can put up a change using the new vars.